.travis.yml

```diff
@@ -1,7 +1,10 @@
 # SPDX-License-Identifier: MIT
 ---
-dist: xenial
```

Comment:
Why is this removed? Xenial seems to be the current default, so it does not change anything now but will once Travis changes the default. We could move to bionic to get Python 3.6 as the system Python removing the need for Python 3.5.

Reply:
It was removed because one reviewer noted that this is default, so it needn't to be noted explicitly. But if intention of explicitness here was to prevent a surprise I will add it back.

Reply:
IMHO it is better to keep it in there to make the choice explicit.